lib/v2/screens/authentication/login_screen.dart

```diff
@@ -9,6 +11,8 @@ import 'package:seeds/v2/navigation/navigation_service.dart';
 
 /// Login SCREEN
 class LoginScreen extends StatelessWidget {
+  static const int approxWidgetHeight = 450;
```

Comment:
Card coding values is not a proper fix. 
@gguij004 please take a look at this.

Reply:
That's the size of the bottom images / widgets, approximately

This code will work as good as any other code you can possibly come up with. 

It will break of course of the bottom design changes, but I don't think there's any solution that would work when the lower design changes.